setup.py

```diff
@@ -7,4 +7,20 @@
     install_requires=[
         'requests~=2.5',
     ],
+    extras_require={
+        "dev": [
```

Comment:
Ah, so this is why we don't need the `dev-requirements.in` anymore? Is this something new or are we using this in other repos? Is this the suggested way to handle dev requirements going forward?

Reply:
I think fewer files in the root is better.

Reply:
I like this convention. Perhaps share this in the feature team channel so others can implement it in their work.

Reply:
Cool, I will remember to mention it when I do a session about parallel testing.